docs/materials/htc/part2-ex1-files.md

```diff
@@ -98,7 +98,7 @@ for word in sorted(words.keys()):
 1.  Do not forget to add a line to name the input file as the argument to the Python script.
 1.  Submit the job, wait for it to finish, and check the output!
 
-If things do not work the first time, keep trying! At this point in the exercises, we are telling you less and less explicitly how to do steps that you have done before. If you get stuck, ask a neighbor or one of the instructors.
+If things do not work the first time, keep trying! At this point in the exercises, we are telling you less and less explicitly how to do steps that you have done before. If you get stuck, ask for help in the Slack channel.
```

Comment:
#general, or should we suggest a specific one like #intro-to-htc?

Reply:
Ah, good point. There's a lot of Slack channels ;) I'll update to #into-to-htc